style/js/shield_defs.js

```diff
@@ -559,6 +559,52 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
```

Comment:
The default generic shield for unsupported networks is also a white rectangle, just like the intended State Highway shield, so this line technically isn’t required.

More importantly, we should make sure as many Texas networks as possible are supported here so that they don’t get confused with State Highways. One that comes to mind is [`network=US:TX:PA`](https://wiki.openstreetmap.org/wiki/Tag%3Anetwork%3DUS%3ATX%3APA) for the one Principal Arterial State System route. Its signs say “PA”, without periods, but periods would be fine too.

Reply:
#213 makes it necessary to explicitly declare white rectangular shields in this file, as you’ve done for state highways.